e2e/__tests__/watchModeNoAccess.test.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'graceful-fs';
+import {cleanup, writeFiles} from '../Utils';
+import {runContinuous} from '../runJest';
+
+const DIR = path.resolve(os.tmpdir(), 'watch_mode_no_access');
+
+const sleep = (time: number) =>
+  new Promise(resolve => setTimeout(resolve, time));
+
+beforeEach(() => cleanup(DIR));
+afterAll(() => cleanup(DIR));
+
+const setupFiles = () => {
+  writeFiles(DIR, {
+    '__tests__/foo.test.js': `
+      const foo = require('../foo');
+      test('foo', () => { expect(typeof foo).toBe('number'); });
+    `,
+    'foo.js': `
+      module.exports = 0;
+    `,
+    'package.json': JSON.stringify({
+      jest: {},
+    }),
+  });
+};
+
+let testRun: ReturnType<typeof runContinuous>;
```

Comment:
don't we export the return type of `runContinuous`? we should

Reply:
Nope, which is weird right? I thought we had an eslint rule or TS config to guard against public exports without explicit typing